src/worker-bindings.cc

```diff
@@ -154,6 +155,48 @@ const char* Worker::InitIsolate() {
   return NULL;
 }
 
+void Worker::WriteCodeCoverageReport() {
+  Locker locker(isolate_);
+  Isolate::Scope isolate_scope(isolate_);
+  Nan::HandleScope scope;
+
+#if NODE_VERSION_AT_LEAST(0, 11, 0)
+  Local<Context> context = Local<Context>::New(isolate_, context_);
+  Context::Scope context_scope(context);
+#else
+  Handle<Context> context = context_;
+  Context::Scope context_scope(context_);
+#endif
```

Comment:
I decided to use a different approach which is IMO simpler and hopefully easier to maintain - see #21. I thought there were three places where this code was duplicated, but as it turns out, there are only two of them :-] I still think the change is worth it.